crates/ruma-client-api/src/backup.rs

```diff
@@ -1,19 +1,19 @@
 //! Endpoints for server-side key backups.
 
-pub mod add_backup_key_session;
-pub mod add_backup_key_sessions;
 pub mod add_backup_keys;
-pub mod create_backup;
-pub mod delete_backup;
-pub mod delete_backup_key_session;
-pub mod delete_backup_key_sessions;
+pub mod add_backup_keys_for_room;
+pub mod add_backup_keys_for_room_session;
```

Comment:
How about

```suggestion
pub mod add_backup_keys_for_room;
pub mod add_backup_keys_for_session;
```

since sessions are always tied to rooms?